src/server/game/Entities/Object/Object.cpp

```diff
@@ -1500,10 +1500,10 @@ float WorldObject::GetGridActivationRange() const
     {
         if (ToPlayer()->GetCinematicMgr()->IsOnCinematic())
             return DEFAULT_VISIBILITY_INSTANCE;
-        return GetMap()->GetVisibilityRange();
+        return GetMap()->GetVisibilityRange() * 5;
```

Comment:
@Palabola yeah, that's just here to show that "it works". Not meant to stay there when (if) this is merged.